packages/react-query/src/__tests__/useQuery.test.tsx

```diff
@@ -6334,7 +6334,7 @@ describe('useQuery', () => {
   it('should respect skipToken and refetch when skipToken is taken away', async () => {
     function Page({ enabled }: { enabled: boolean }) {
       const { data, status } = useQuery({
-        queryKey: ['key'],
+        queryKey: ['key-skip-token'],
```

Comment:
ah I missed this. If this works, please use the provided helper function to create unique keys:

https://github.com/TanStack/query/blob/5e648737524e18121213be243f791721c309a4b4/packages/react-query/src/__tests__/useQueries.test.tsx#L843